src/Kirel.Blazor.Logger/Program.cs

```diff
@@ -18,9 +18,40 @@
 builder.RootComponents.Add<App>("#app");
 builder.RootComponents.Add<HeadOutlet>("head::after");
 
+var defaultBaseUri = new Uri(builder.HostEnvironment.BaseAddress);
+var hostOnlyBaseStr =  $"{defaultBaseUri.Scheme}://{defaultBaseUri.Host}:{defaultBaseUri.Port}/";
+
+<<<<<<< Updated upstream
 var identityUriStr = builder.Configuration["Services:Identity"];
 var httpLogsStrUri = builder.Configuration["Services:HttpLogsApi"];
```

Comment:
rename pls to httpLogsUriStr and next line too.

Reply:
done